.github/workflows/build_and_cache_Kokkos_linux.yml

```diff
@@ -17,10 +17,6 @@ on:
         description: "Kokkos version"
         value: ${{ jobs.linux-set-builder-matrix.outputs.kokkos_version }}
 
-concurrency:
-  group: build_and_cache_Kokkos_linux-${{ github.ref }}-${{ github.workflow }}
-  cancel-in-progress: true
-
```

Comment:
Are we needing to remove this concurrency group?

Reply:
Yes, because this is now called by both the C++ and Python workflows.